lib/ansible/modules/network/panos/panos_import.py

```diff
@@ -47,13 +50,20 @@
     category:
         description:
             - Category of file uploaded. The default is software.
+            - See API > Import section of the API reference for category options.
         default: software
     file:
         description:
             - Location of the file to import into device.
     url:
         description:
             - URL of the file that will be imported to device.
+    validate_cert:
+        description:
+            - If C(no), SSL certificates will not be validated. This should only set to no used on personally controlled sites using self-signed certificates.
```

Comment:
"This should only set to no used on" - sounds awkward.  I also don't recommend reasons for when to turn this off.  I'd just warn them it's not recommend.   Even personally controlled sites using self-signed certs can fall victim to MITM attacks.